assets/xml/objects/object_fish.xml

```diff
@@ -1,161 +1,180 @@
+        <Limb Name="gFishingFishEndRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_END_ROOT" Offset="0x28F4" />
+        <Limb Name="gFishingFishTailFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TAIL_FIN" Offset="0x2900" />
+        <Limb Name="gFishingFishCaudalPeduncleEndLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE_END" Offset="0x290C" />
+        <Limb Name="gFishingFishCaudalPeduncleLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE" Offset="0x2918" />
+        <Limb Name="gFishingFishTopRearFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TOP_REAR_FIN" Offset="0x2924" />
+        <Limb Name="gFishingFishBottomRearFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_BOTTOM_REAR_FIN" Offset="0x2930" />
+        <Limb Name="gFishingFishTorsoLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TORSO" Offset="0x293C" />
+        <Limb Name="gFishingFishTopFrontFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TOP_FRONT_FIN" Offset="0x2948" />
+        <Limb Name="gFishingFishBottomFrontFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_BOTTOM_FRONT_FIN" Offset="0x2954" />
+        <Limb Name="gFishingFishJawLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_JAW" Offset="0x2960" />
+        <Limb Name="gFishingFishRightPectoralFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_RIGHT_PECTORAL_FIN" Offset="0x296C" />
+        <Limb Name="gFishingFishLeftPectoralFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_LEFT_PECTORAL_FIN" Offset="0x2978" />
         <Skeleton Name="gFishingFishSkel" Type="Flex" LimbType="Standard" LimbNone="FISHING_FISH_LIMB_NONE" LimbMax="FISHING_FISH_LIMB_MAX" EnumName="FishingFishLimb" Offset="0x29C0" />
-        <Texture Name="object_fish_Tex_0029D0" OutName="tex_0029D0" Format="rgba16" Width="8" Height="16" Offset="0x29D0" />
-        <Texture Name="object_fish_Tex_002AD0" OutName="tex_002AD0" Format="rgba16" Width="8" Height="8" Offset="0x2AD0" />
-        <DList Name="object_fish_DL_002BD0" Offset="0x2BD0" />
-        <Texture Name="object_fish_Tex_002CE0" OutName="tex_002CE0" Format="i4" Width="32" Height="64" Offset="0x2CE0" />
-        <Texture Name="object_fish_Tex_0030E0" OutName="tex_0030E0" Format="i4" Width="16" Height="16" Offset="0x30E0" />
+
+        <Texture Name="gFishingOldLureHookTex" OutName="fishing_old_lure_hook" Format="rgba16" Width="8" Height="16" Offset="0x29D0" />
+        <Texture Name="gFishingOldLureFloatTex" OutName="fishing_old_lure_float" Format="rgba16" Width="8" Height="8" Offset="0x2AD0" />
+        <DList Name="gFishingOldLureDL" Offset="0x2BD0" />
+
+        <Texture Name="gFishingStreamSplashTile1Tex" OutName="fishing_stream_splash_tile_1" Format="i4" Width="32" Height="64" Offset="0x2CE0" />
+        <Texture Name="gFishingStreamSplashTile2Tex" OutName="fishing_stream_splash_tile_2" Format="i4" Width="16" Height="16" Offset="0x30E0" />
```

Comment:
```suggestion
        <-- Old Lure -->
        
        <-- Old Lure Textures -->
        <Texture Name="gFishingOldLureHookTex" OutName="fishing_old_lure_hook" Format="rgba16" Width="8" Height="16" Offset="0x29D0" />
        <Texture Name="gFishingOldLureFloatTex" OutName="fishing_old_lure_float" Format="rgba16" Width="8" Height="8" Offset="0x2AD0" />
        
        <-- Old Lure DL -->
        <DList Name="gFishingOldLureDL" Offset="0x2BD0" />

        <-- Stream Splash -->
        
        <-- Stream Splash Textures -->
        <Texture Name="gFishingStreamSplashTile1Tex" OutName="fishing_stream_splash_tile_1" Format="i4" Width="32" Height="64" Offset="0x2CE0" />
        <Texture Name="gFishingStreamSplashTile2Tex" OutName="fishing_stream_splash_tile_2" Format="i4" Width="16" Height="16" Offset="0x30E0" />
```
Could you seperate things into categories? I would actually have major categories for each thing (since this objects contains assests for multiple things) and than the sub categories for each asset type.

Reply:
small issue tho, the owner dls and limbs are separated from the textures by some other stuff so idk breaks the major category

Reply:
That is fine. I might even add another `<-- Owner -->` section for the second part, but I'll leave it up to you.